### ISSUE TITLE
Feature Request: Option to Invert Images

### ISSUE THREAD

**Issue by vicerious001** [Midnight-Lizard/Midnight-Lizard#333]
It would be nice to have a checkbox or similar to invert the images on a page. This would be helpful on pages with lots of graphs/schematics, and also for sites like the black-and-white browser game Kingdom of Loathing.

Maybe this could be done through the CSS invert function?

**Comment by pavel-agarkov** [comment]
There is already similar option: `Do not invert dynamic content` on the Common tab
Have you tried it with Kingdom of Loathing?

**Comment by vicerious001** [comment]
Yes, I tried that option on Kingdom of Loathing. It made no difference.
